static/app/views/onboarding/setupDocs.tsx

```diff
@@ -432,6 +447,14 @@ function SetupDocs({search, route, router, location, ...props}: Props) {
               project={project}
               location={location}
             />
+          ) : showLoaderOnboarding ? (
+            <SetupDocsLoader
+              organization={organization}
+              project={project}
+              location={location}
+              platform={loadedPlatform}
+              close={() => setShowLoaderOnboarding(false)}
```

Comment:
I like this solution, but we might consider saving it to localStorage in the future for the case users decide to reload the page